param.go

```diff
@@ -68,8 +68,11 @@ func (e *engine) parseRequestParams(ctx *Context) {
 		}(req)
 	}
 
-	// i18n option override by Query parameter `lang`
-	if lang := ctx.Req.QueryValue(keyOverrideLocale); !ess.IsStrEmpty(lang) {
+	// i18n option via the config value "i18n.url_param_name".
+	//If that value is missing, we default to the `lang` query parameter
+	queryParam := AppConfig().StringDefault("i18n.url_param_name", keyOverrideLocale)
+
```

Comment:
Sorry about that.. I am used to that (for readability), would take a while to get used too. Would fix